ext/opcache/config.m4

```diff
@@ -219,7 +219,11 @@ int main() {
   char *shm;
   char tmpname[4096];
 
+#ifndef __FreeBSD__
   sprintf(tmpname,"test.shm.%dXXXXXX", getpid());
+#else
+  sprintf(tmpname,"/tmp/test.shm.%dXXXXXX", getpid());
```

Comment:
I think it would make sense to just use this for everything, not just FreeBSD. I'm assuming that the previous code created this file in the current working directory, which seems like a not so good idea.

Reply:
I thought to do that at first but did not dare for other systems than Linux for example.